src/components/HotspotListItem.tsx

```diff
@@ -32,13 +33,32 @@ const HotspotListItem = ({
     const nextReward = await hntBalanceToDisplayVal(totalReward, false)
     setReward(`+${nextReward}`)
   }, [hntBalanceToDisplayVal, totalReward])
+  const { blockHeight } = useSelector((state: RootState) => state.heliumData)
```

Comment:
An small optimization would be to write this like...
`const blockHeight = useSelector((state: RootState) => state.heliumData.blockHeight)`

Reply:
Updated